pavelib/servers.py

```diff
@@ -89,6 +90,8 @@ def devstack(args):
     parser = argparse.ArgumentParser(prog='paver devstack')
     parser.add_argument('system', type=str, nargs=1, help="lms or studio")
     parser.add_argument('--fast', action='store_true', default=False, help="Skip updating assets")
+    parser.add_argument('--optimized', action='store_true', default=False, help="Run with optimized assets")
+    parser.add_argument('--settings', type=str, default="devstack", help="Settings file")
```

Comment:
I was thinking of ``asset_settings`` as an internal option that I used to implement the optimized mode. However, it makes sense to support it externally too, so I've added it to all the server methods.